_tools/packaging/prepare-deb-build.sh

```diff
@@ -7,6 +7,7 @@ pwd=`dirname $0`
 
 MACKEREL_AGENT_NAME=${MACKEREL_AGENT_NAME:-mackerel-agent}
 MACKEREL_AGENT_VERSION=$(grep -o -e "[0-9]\+.[0-9]\+.[0-9]\+-[0-9]" packaging/deb/debian/changelog | head -1 | sed 's/-.*$//')
+BUILD_DIRECTORY=${BUILD_DIRECTORY:-build/}
```

Comment:
If BUILD_DIRECTORY is not specified, it is set `build/` here and this line 20 becomes `cd "build//mackerel-agent" "..."`, am I correct?